docs/source/guides/getting-started/installing-idom.rst

```diff
@@ -1,28 +1,28 @@
 Installing IDOM
 ===============
 
-Installing IDOM with ``pip`` will generally require doing so alongside a supported
-server implementation. This can be done by specifying an installation extra using square
-brackets. For example, if we want to run IDOM using `Starlette
-<https://www.starlette.io/>`__ we would run:
+Installing IDOM with ``pip`` will typically require doing so alongside a built-in
+backend implementation. This can be done by specifying an installation extra using
+square brackets. For example, if we want to run IDOM using the `Starlette
+<https://www.starlette.io/>`__ backend we would run:
 
 .. code-block:: bash
 
     pip install "idom[starlette]"
 
-If you want to install a "pure" version of IDOM without a server implementation you can
-do so without any installation extras. You might do this if you wanted to user a server
-which is not officially supported or if you wanted to manually pin your dependencies:
+If you want to install a "pure" version of IDOM without a backend implementation you can
+do so without any installation extras. You might do this if you wanted to use a backend
+which does not have built-in support or if you wanted to manually pin your dependencies:
 
 .. code-block:: bash
 
     pip install idom
 
 
-Officially Supported Servers
-----------------------------
+Built-in Backends
```

Comment:
I was worried this might show some preference over frameworks integrations that are distributed separately like Django IDOM. "Native Backends" seems alright, but the others make it seem like those solutions are not "supported" or "official".

Reply:
How about "Natively Supported Backends"?